### DISCUSSION TITLE
Oxalis-AS4 5.2.0 Release

### DISCUSSION THREAD

**Discussion by aaron-kumar** [OxalisCommunity/Oxalis-AS4#186] [General]
- [Neethi version upgrade](https://github.com/OxalisCommunity/Oxalis-AS4/commit/a8855cc94221a8d8c363c823f8f5483eaac78b54)
- [wss4j bug fix](https://github.com/OxalisCommunity/Oxalis-AS4/commit/0e7ddbc4e522360b5e9013aac6b481946079b5af)
- [Task 163: CXF version upgrade - Security vulnerability](https://github.com/OxalisCommunity/Oxalis-AS4/commit/739a7e08e87c823bfa77cd60e3fbcd9da1e689ba)
- [Upgraded to Oxalis version 5.2.0](https://github.com/OxalisCommunity/Oxalis-AS4/commit/80d27c4b95a2ffedf8f30698783e53fb361f78f6)

<hr /><em>This discussion was created from the release <a href='https://github.com/OxalisCommunity/Oxalis-AS4/releases/tag/v5.2.0'>Oxalis-AS4 5.2.0 Release</a>.</em>

**Comment by HegeErdal** [comment]
After upgrading to this version, I'm seeing a lot of warnings like this:

org.apache.jcp.xml.dsig.internal.dom.DOMReference: 
"The input bytes to the digest operation are null. This may be due to a problem with the Reference URI or its Transforms."

I'm not sure exactly which library-update caused this, or if it should be of a concern to us, but please have a look.
